README.md

```diff
@@ -24,6 +24,24 @@ Run
 
 This local development includes hot reloading on the front-end and the back-end.
 
+Make sure that your have the necessary environment varibales available in the `.env` file and the `docker-compose.yml` file
```

Comment:
typo "varibales"
please check this section compatibility with my readme

Reply:
I will check this.